data/sql/plugins.qtAccessionPlugin.lib.model.schema.sql

```diff
@@ -77,6 +77,58 @@ CREATE TABLE `accession_i18n`
 		ON DELETE CASCADE
 )Engine=InnoDB;
 
+#-----------------------------------------------------------------------------
+#-- accession_event
+#-----------------------------------------------------------------------------
+
+DROP TABLE IF EXISTS `accession_event`;
+
+
+CREATE TABLE `accession_event`
```

Comment:
The event model represents a date range, rather than just a single date, and agents aren't necessarily AtoM actors or users. I think Steve also came to the conclusion that it wouldn't be good to try to use the existing event table.

Reply:
Hmm, I do wonder if it's a good idea to make the agent column a varchar rather than a related entity like authority records, donors, and rights holders. 🤔 There are pros and cons with either design of course, but I'm not clear why agent doesn't follow the established pattern.

Anyway, I don't consider it a blocker if you and Steve decided a new table is necessary, and agent should be varchar column.